.github/workflows/verify_library_generation.yaml

```diff
@@ -0,0 +1,56 @@
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+    paths:
+    - library_generation/**
+
+  workflow_dispatch:
+name: verify_library_generation
+jobs:
+  verify_library_generation:
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        java: [ 8 ]
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-java@v3
+      with:
+        java-version: ${{ matrix.java }}
+        distribution: temurin
+        cache: maven
+    - name: checkout googleapis
+      uses: actions/checkout@v3
+      with:
+        repository: googleapis/googleapis
+        ref: 90a551c2a02613507a2f0f05fd25d133113a1a1c
+        path: googleapis
+    - name: copy protos to proto_path
+      run: |
+        cp -r googleapis/google library_generation
+    - name: generate client library
```

Comment:
Not sure if we have an existing pattern to test shell scripts, can we have something like `generate_library_test.sh` that have all the test cases? So that we can test more scenarios and it can be run in local as well? +@meltsufin

Reply:
As discussed in the meeting, the testing script will be in future PRs.

Reply:
I changed this action to run `generate_library_integration_test.sh`. The test script can run locally as well.

I'll resolve this comment, please reopen it if necessary.